src/components/Categories.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+const Categories = () => (
+  <div>
+    <button type="button">Category</button>
+  </div>
+);
```

Comment:
- [ ] The button for this component is supposed to display the text "Check Status" as required in the project requirement. Kindly change the 'Category' text in the button to "Check Status". 👍